o.apache.xml.resolver/external/binaries-list

```diff
@@ -14,4 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-B0D0FCBAC68826D2AFA3C7C89FC4D57B95A000C3 resolver-1.2.jar
+#
+# We preserve the following line for historic reasons
+# B0D0FCBAC68826D2AFA3C7C89FC4D57B95A000C3 resolver-1.2.jar
```

Comment:
I would think about just removing this. Lets assume in one year the netbeans binaries server is gone, then there is no way to access the binary (maybe this is one of the major advantage of using maven central).